src/map/constants.inc.h

```diff
@@ -0,0 +1,970 @@
+	script->constdb_comment("Status Icons");
```

Comment:
extraction constants need do in separate commit too. They not related to clan system at all